config/ant-phase-verify.xml

```diff
@@ -53,6 +53,8 @@
                 file="config/suppressions.xml"/>
       <property key="checkstyle.suppressions-xpath.file"
                 file="config/suppressions-xpath.xml"/>
+      <property key="checkstyle.java.version"
```

Comment:
checkstyle.source.java.version

Source are on syntax of jdk11, but actual jdk that compiled can be jdk21.

Reply:
Ok, here we define target jdk https://github.com/checkstyle/checkstyle/blob/6a3ab019020cf67973e4f3fdc8ace6d197eadddc/pom.xml#L224

Reply:
Actually, let's keep it as is